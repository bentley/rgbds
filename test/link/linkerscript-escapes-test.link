ROM0
	"A\"B\tC\rD\nE"
	"in\{valid"
