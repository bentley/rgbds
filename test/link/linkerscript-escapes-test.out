error: ./linkerscript-escapes-test.link(3): Illegal character escape
